server/render/index.js

```diff
@@ -73,20 +65,29 @@ export function render( element, key = JSON.stringify( element ) ) {
 
 export function serverRender( req, res ) {
 	const context = req.context;
+	let title, metas = [], links = [];
 
 	if ( context.lang !== config( 'i18n_default_locale_slug' ) ) {
 		context.i18nLocaleScript = '//widgets.wp.com/languages/calypso/' + context.lang + '.js';
 	}
 
-	if ( config.isEnabled( 'server-side-rendering' ) &&
-		context.store &&
-		context.layout &&
-		! context.user ) {
-		context.initialReduxState = pick( context.store.getState(), 'ui', 'themes' );
-		const key = context.renderCacheKey || JSON.stringify( context.layout );
-		Object.assign( context, render( context.layout, key ) );
+	if ( config.isEnabled( 'server-side-rendering' ) && ! context.user ) {
+		if ( context.layout ) {
```

Comment:
Interesting that the `context.layout` and `context.store` branches have been split apart here, because I don't thing we ever want to do one without the other?